### PR TITLE
Add docs for accessing live Dalmatian environments

### PR DESCRIPTION
**Pull request by pezholio**
Adding some documentation for accessing the containers in Dalmatian, as well as some docs on how to get a copy of  prod/staging content to get a realistic(ish) development environment.

### CODE REVIEW

File: docs/console-access.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Console Access
+
+_Note_ The below assumes you have access to the dxw Dalmatian infrastructure,
+and have `dalmatian-tools` installed. If you don't have access, and need to,
+contact a member of the dxw ops team.
+
+## Login to dalmatian
+
+```bash
```
Comment (rjw1) [inline docs/console-access.md:9]: you should only need to do this for initial setup.

**Review by rjw1 — CHANGES_REQUESTED** [review]
One minor change around `dalmatian login` but otherwise looks good.

Personally I would make the variables actually have a `$` when using them in a command so it just looks more shellish and obvious needs to be either set or replaced with text.

File: docs/restoring-dump-from-live.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+docker ps
+```
+
+Copy the file from the relevant container:
+
+```bash
+docker cp CONTAINER_NAME:/tmp/$(date +%Y%m%d).json /tmp/$(date +%Y%m%d).json
+```
+
+Upload the file to S3 (where `ENVIRONMENT` is one of `staging` or `prod`)
```
Comment (rjw1) [inline docs/restoring-dump-from-live.md:54]: I actually have a working file download command which I'm going to put in a PR for tomorrow.

Comment (dragon-dxw) [reply docs/restoring-dump-from-live.md:54]: What's the status of this file download command, and where can I learn more?

**Pull request merged**